alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -161,7 +164,7 @@ abstract class BaseAlertPresenter(private val alert: Alert) : AlertActions {
          * @param handler The block to execute after user taps a button
          */
         fun setPositiveButton(title: String, handler: AlertActionHandler = {}) = apply {
```

Comment:
There is no way to add action with custom color using builder? (You can create Action but can't use within Alert)